app/services/word_service.rb

```diff
@@ -1,16 +1,20 @@
+require './lib/modules/serviceable'
 class WordService
   def self.conn
    response =  Faraday.new("https://shielded-headland-00998.herokuapp.com")
   end
 
   def self.word_search
     response = conn.get("/api/v1/words?amount=")
-    JSON.parse(response.body, symbolize_names: true) 
+    # JSON.parse(response.body, symbolize_names: true) 
+    require 'pry'; binding.pry
+    Serviceable.parse_json(response)
   end
 
   def self.multi_word(num=4)
```

Comment:
should this argument be updated since the controller is now establishing a default value... and passing it down?

Reply:
completed, see [a845056](https://github.com/Correct-or-Crass/Trivia_API/pull/41/commits/a8450560481585e0b01dc617f7c9720e7b636273) below